### PR TITLE
Daniil2

### PR DESCRIPTION
**Pull request by Dablup**
Added model of the club, created a template for the page of the club, created reset_password(but we need Gmail account)

### CODE REVIEW

File: InnoClubs/InnoClubs/apps/clubs/views.py [hunk 1]
```diff
@@ -1,11 +1,33 @@
...
+# if user is not logged in => redirect to the auth/
+# if user is logged in => show main page
+# args['infoFromDb'] = .... means that we add information of all the clubs from database to the args
 def main(request):
     args = {}
     if auth.get_user(request).username:
         args['username'] = auth.get_user(request).username
+        try:
+            args['infoFromDb'] = Club.objects.all()
+        except:
```
Comment (bragov4ik) [inline InnoClubs/InnoClubs/apps/clubs/views.py]: https://www.flake8rules.com/rules/E722.html

File: InnoClubs/InnoClubs/apps/clubs/views.py [hunk 2]
```diff
@@ -1,11 +1,33 @@
...
+# if user is not logged in => redirect to the auth/
+# if user is logged in => show main page
+# args['infoFromDb'] = Club.objects.get(club_url=club_url) means that we add information of the exactly one club that was accessed
+def ClubPage(request, club_url):
+    args = {}
+    if auth.get_user(request).username:
+        args['username'] = auth.get_user(request).username
+        try:
+            args['infoFromDb'] = Club.objects.get(club_url=club_url)
+        except:
```
Comment (bragov4ik) [inline InnoClubs/InnoClubs/apps/clubs/views.py]: https://www.flake8rules.com/rules/E722.html

Comment (Dablup) [reply InnoClubs/InnoClubs/apps/clubs/views.py]: fixed

**Review by bragov4ik — APPROVED** [review]
It's working for me

**Pull request merged**